junit5-engine/src/main/java/org/junit/gen5/engine/junit5/execution/ConditionEvaluator.java

```diff
@@ -108,4 +137,42 @@ private void logResult(Class<?> conditionType, ConditionEvaluationResult result)
 		LOG.finer(() -> String.format("Evaluation of condition [%s] resulted in: %s", conditionType.getName(), result));
 	}
 
+	private Predicate<Object> conditionIsActivated(ConfigurationParameters configurationParameters) {
+		Pattern deactivatePattern = getDeactivatePattern(configurationParameters);
+		if (deactivatePattern != null) {
+			if (DEACTIVATE_ALL_CONDITIONS_PATTERN.equals(deactivatePattern)) {
```

Comment:
Right again!

This was the result of some refactoring, and unfortunately the tests continued to pass.

Thanks for pointing this out.

I'll fix it.